types/d3-shape/index.d.ts

```diff
@@ -2395,24 +2393,34 @@ export interface Symbol<This, Datum> {
 }
 
 /**
- * Constructs a new symbol generator with the default settings.
- */
-export function symbol(): Symbol<any, any>; // tslint:disable-line ban-types
-
-/**
- * Constructs a new symbol generator with the default settings.
+ * Constructs a new symbol generator of the specified type and size.
+ * If not specified, type defaults to a circle, and size defaults to 64.
  *
- * The generic corresponds to the data type of the datum underlying the symbol.
+ * The first generic corresponds to the "this" context within which the symbol generator is invoked.
+ * The second generic corresponds to the data type of the datum underlying the symbol.
+ *
+ * @param type The specified type.
+ * @param size The specified size.
  */
-export function symbol<Datum>(): Symbol<any, Datum>; // tslint:disable-line ban-types
+export function symbol<Datum = any>(
+    type?: SymbolType | ((this: any, d: Datum, ...args: any[]) => SymbolType),
+    size?: number | ((this: any, d: Datum, ...args: any[]) => number)
+): Symbol<any, Datum>; // tslint:disable-line ban-types
```

Comment:
Why was a suppression required here?

Reply:
I didn't add this code. I'll check to see if it can be removed.

Reply:
It wasn't necessary. I've removed it.